dot-net-sdk/validators/RuleValidator.cs

```diff
@@ -25,40 +26,73 @@ private static List<bool> EvaluateRuleCondition(SubjectAttributes subjectAttribu
 
     private static bool EvaluateCondition(SubjectAttributes subjectAttributes, Condition condition)
     {
-        if (subjectAttributes.ContainsKey(condition.attribute))
+        try
         {
-            subjectAttributes.TryGetValue(condition.attribute, out EppoValue outVal);
-            var value = outVal!;
-            Dictionary<OperatorType, Func<EppoValue, EppoValue, bool>> validationFunctions = new()
+            if (subjectAttributes.ContainsKey(condition.attribute) &&
+                subjectAttributes.TryGetValue(condition.attribute, out EppoValue outVal))
             {
-                { GTE, (a, b) => a.DoubleValue() >= b.DoubleValue() },
-                { GT, (a, b) => a.DoubleValue() > b.DoubleValue() },
-                { LTE, (a, b) => a.DoubleValue() <= b.DoubleValue() },
-                { LT, (a, b) => a.DoubleValue() < b.DoubleValue() },
+                var value = outVal!; // Assuming non-null for simplicity, handle nulls as necessary
```

Comment:
I'm not super happy about this - however it was existing and the `try/catch` should handle it